client/src/components/HostDescription.test.jsx

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import { render, screen } from '@testing-library/react';
+import HostDescription from './HostDescription';
+
+describe('HostDescription', () => {
+
+  test('should render all elements when all props are provided or truthy', () => {
+
+    const props = {
+      bio: 'It was the best of times, it was the worst of times ...',
+      cohosts: [
+        {
+          avatarUrl: 'http://placekitten.com/75/75',
+          name: 'Jodi'
+        }
+      ],
+      duringYourStay: 'Beware of dog',
+      isSuperhost: true,
+      name: 'Ted'
+    };
+
+    const { getByText } = render(<HostDescription {...props} />);
```

Comment:
is getByText used anywhere?

Reply:
I guess I forgot to write some expect() statements. Nice catch.